docs/Concepts/PreparingForTheMerge.md

```diff
@@ -0,0 +1,63 @@
+# Preparing for the Merge
+
+## Overview
+'The Merge' is the process where Ethereum changes from a proof of work network, to a proof of stake consensus network.
+
+It's been described anecdotally as 'switch engines in the car from gas to electric while going 90 down the freeway'.
+
+## What to expect
+The consensus layer client (previously eth2 client, eg. Teku, Lighthouse, Prysm, Nimbus, loadstar) will have configuration defined to instruct it to start listening for a certain terminal difficulty to be reached on the eth1 endpoint.
+
+Once that difficulty is reached, eth1 will then become the execution layer of Ethereum, and the consensus layer will be responsible for progressing the chain.
+
+The consensus layer will enable Bellatrix (the merge configuration) prior to the terminal difficulty being reached, then it is up to eth1 blocks to reach the pre-determined difficulty, then from that point Ethereum moves to a proof of stake network.
+
+Post merge, as well as validators earning rewards for performing duties, there are also earnings from the execution engine performing work, and these earnings are directed to a fee recipient address.
+
+## Execution Engine
+
+Pre-merge, an eth1-client is required for block proposal to get deposits for validators. They're intermittent, and you get the data from other blocks if your eth1 client is offline for some reason.
+
+Once the merge is complete, the execution engine becomes much more heavily relied upon for execution. It will be no longer viable for an external provider to be referenced, each beacon-node will need to be closely associated with an execution engine (eth1 client).
+
+Each beacon-node will require its own execution-engine, so if you are maintaining multiple beacon-nodes, you will need to set up an execution-engine for each beacon-node. Service providers that provide eth1 access will not be adequate for a beacon-node to continue to function on the network.
```

Comment:
Also worth noting that it isn't currently possible to have a beacon-node use a second execution client as a backup.  It's a 1-1 mapping between beacon node and execution client.